styles.css

```diff
@@ -0,0 +1,4 @@
+
+h1 {
+  color: cadetblue;
+}
```

Comment:
- [ ] [OPTIONAL] It's always good practice to use classes instead of HTML elements.